src/Moryx.AbstractionLayer/Tasks/TaskStep.cs

```diff
@@ -66,6 +67,23 @@ private static OutputDescription[] DescriptionsFromEnum(Type enumType)
                     }).ToArray();
         }
 
+        /// <summary>
+        /// Get the enum name. If Display attribute is applied on the enum value
+        /// the display value get used otherwise the enum value is used.
+        /// </summary>
+        /// <param name="value"> value of the enum</param>
+        /// <param name="enumType">type of the enum</param>
+        /// <returns></returns>
+        private static string GetEnumName(object value, Type enumType)
+        {
+            var enumMembers = enumType.GetMembers();
+            var enumValueInfo = enumMembers?.FirstOrDefault(x => x.DeclaringType == enumType && 
```

Comment:
Please don't use null checks for every invocation if the argument should not be null. Each `?` adds 3 invisible lines to your source code.